packages/ui-kit/src/components/TimeRangePicker/DateTimeField/DateTimeField.tsx

```diff
@@ -0,0 +1,216 @@
+import {
+  addDays,
+  addMinutes,
+  format as dateFormat,
+  isAfter,
+  isBefore,
+  isValid,
+  parse,
+  subDays,
+  subMinutes
+} from 'date-fns'
```

Comment:
Let's try to refactor this to use luxon instead if possible

Reply:
I would prefer to stick with `date-fns` for now since `react-day-picker` is based on it. Otherwise we may face issues converting stuff from `luxon` to `date-fns`. Do you see any downsides to using `date-fns` in this particular component?

Reply:
The main downside is having two libraries for date management, also keep in mind that `date-fns` adds up a [considerable size to the bundle](https://www.npmjs.com/package/date-fns).

`luxon` works better with timezones and requires no dependencies (`date-fns` would require `date-fns-tz` for this), if you check the `TimeSeries` dependencies, you'll see that we are already handling this with `luxon`, previously we used `date-fns` but decided to move away from it as it was also causing issues with the chart.js adapter.

Reply:
Keeping in mind that `react-day-picker` requires it as a peer dependency let's just use it here for now